.gitignore

```diff
@@ -14,4 +14,6 @@ mirage/*/tls_server
 Makefile
 key_gen.ml
 
-random/
\ No newline at end of file
+random/
+vendor
```

Comment:
This is needed if we want to develop against vendored dependencies.

Reply:
please remove from this PR. you can open a separate PR with the "develop against vendored dependencies" goal.

Reply:
Done in https://github.com/mirleft/ocaml-tls/pull/462